var/spack/repos/builtin/packages/netcdf/package.py

```diff
@@ -22,9 +22,13 @@ class Netcdf(AutotoolsPackage):
     and sharing of array-oriented scientific data."""
 
     homepage = "http://www.unidata.ucar.edu/software/netcdf"
-    url      = "http://www.gfd-dennou.org/arch/netcdf/unidata-mirror/netcdf-4.3.3.tar.gz"
+    git      = "https://github.com/Unidata/netcdf-c"
+    url      = "https://github.com/Unidata/netcdf-c/archive/v4.6.2.tar.gz"
```

Comment:
If you switch to the GitHub tarballs, you'll have to update the checksums of all previous releases as well.

Reply:
I went back to the original url since spack (or at least my use of it) could not find the older versions when using the GitHub tarballs even though they are there...